crts/microsoft/store/x64/x64-14.29.30037.json

```diff
@@ -0,0 +1,42 @@
+{
+  "contacts": {
+    "Mark Levine": {
+      "email": "markle@microsoft.com",
+      "role": [
+        "publisher"
+      ]
+    }
+  },
+  "demands": {
+    "windows": {
```

Comment:
It is better to remove the "demands" in the "dependency only" artifact as they are handled in the referencing "public" artifact.